File: LeetCode/fizz-buzz.py
from typing import List

class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        mylist = []
        for num in range(1, n+1):
            output = ""
            if num % 3 == 0:
                output += 'Fizz'
            if num % 5 == 0:
                output += 'Buzz'
            if not output:
                output = str(num)
            mylist.append(output)
        return mylist
if __name__ == "__main__":
    solution  = Solution()
    n = 3
    result = solution.fizzBuzz(n)
    print(result)

File: Leetcode Javascript/fizz-buzz.js
var fizzBuzz = function(n) {
    let my_array = []
    for (i = 1; i <= n; i++) {
        let output = ""
        if (i % 3 === 0) {
            output += 'Fizz'
        }
        if (i % 5 === 0) {
            output += 'Buzz'
        } 
        if (output === "") {
            output = String(i)
        }
        my_array.push(output)
        
    }
    return my_array
};

console.log(fizzBuzz(5))